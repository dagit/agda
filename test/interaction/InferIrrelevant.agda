-- Andreas, 2011-10-04
-- I'd like to infer the type of a even though it is irrelevant
-- E.g. when pressing C-c C-.
module InferIrrelevant where

f : (A : Set)(g : .A → A).(a : A) → A
f A g a = {!a!}