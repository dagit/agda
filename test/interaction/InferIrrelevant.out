(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-info-action "*Type-checking*" "Checking InferIrrelevant (InferIrrelevant.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished InferIrrelevant.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : A\n" nil)
(agda2-goals-action '(0))
(agda2-status-action "")
(agda2-info-action "*Inferred Type*" "A" nil)
(agda2-goals-action '(0))
